ui/src/app/kvm/views/KVMList/StorageColumn/StorageColumn.tsx

```diff
@@ -18,9 +18,12 @@ const StorageColumn = ({ id }: Props): JSX.Element | null => {
   );
 
   if (pod) {
-    const availableStorage = formatBytes(pod.total.local_storage, "B");
-    const allocatedStorage = formatBytes(pod.used.local_storage, "B", {
-      convertTo: availableStorage.unit,
+    const { allocated_other, allocated_tracked, free } = pod.resources.storage;
+    const totalInBytes = allocated_other + allocated_tracked + free;
+    const totalStorage = formatBytes(totalInBytes, "B", { decimals: 1 });
+    const allocatedStorage = formatBytes(allocated_tracked, "B", {
```

Comment:
It seems like we're doing nearly the same in StorageResources, is this another case where we could use a util or hook or is it not quite similar enough?

Reply:
Yeah we could definitely use a hook or util, but I think I'll handle that in https://github.com/canonical-web-and-design/maas-ui/issues/2568 rather than here.